data/phi1035/phi001/__cts__.xml

```diff
@@ -1,9 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi1035" projid="latinLit:phi001" urn="urn:cts:latinLit:phi1035.phi001" xml:lang="lat">
-            <ti:title xml:lang="eng">Argonautica</ti:title>
+            <ti:title xml:lang="lat">Argonautica</ti:title>
             <ti:edition projid="latinLit:perseus-lat2" urn="urn:cts:latinLit:phi1035.phi001.perseus-lat2" workUrn="urn:cts:latinLit:phi1035.phi001">
-                <ti:label xml:lang="eng">Argonautica</ti:label>
-                <ti:description xml:lang="eng">C. Valerius Flaccus. Otto Kramer. C. Valeri Flacci
-                    Setini Balbi Argonauticon Libri Octo. Leipzig. Teubner. 1913.</ti:description>
+                <ti:label xml:lang="lat">Argonautica</ti:label>
+                <ti:description xml:lang="mul">Valerius Flaccus. Kramer, Otto, editor. C. Valeri Flacci
```

Comment:
We usually do [primary author.] [frontpiece book title.] [editor/translator.] [pub place: publisher, year.]
It looks like the editor is out of order here.

Reply:
I will fix that.